Improve introduction to Notebooks section (#81)
SoftDev4Research/4OSS-lesson: _episodes/02-make-it-public.md

### What is Jupyter?

> Jupyter notebook, formerly known as IPython Notebook, is a flexible and powerful open source research tool that can help you keep a narrative of your coding process. The name Jupyter is an acronym of the first three languages that Notebookd supported: **JU**lia, **PYT**hon, and **R**. [Project Jupyter](http://jupyter.org/) can be used for interactive data science and scientific computing in more over 40 programming languages. There are a lot of resources that can help you better understand how to prepare and structure a Jupyter notebook, such as the [official Documentation](http://jupyter.org/documentation) and the [Introduction materials for Reproducible Research Curriculum](https://github.com/Reproducible-Science-Curriculum/introduction-RR-Jupyter).

Reviewer: ```suggestion
Jupyter notebook, formerly known as IPython Notebook, is a flexible and powerful open source research tool that can help you keep a narrative of your coding process. The name Jupyter is an acronym of the first three languages that this Notebook supported: **JU**lia, **PYT**hon, and **R**. [Project Jupyter](http://jupyter.org/) can be used for interactive data science and scientific computing. Later it supported over 40 other programming languages. There are a lot of resources that can help you better understand how to prepare and structure a Jupyter notebook, such as the [official Documentation](http://jupyter.org/documentation) and the [Introduction materials for Reproducible Research Curriculum](https://github.com/Reproducible-Science-Curriculum/introduction-RR-Jupyter).
```

> Jupyter is one type of computational notebooks. Another type that shares the same philosophy and overall structure (but has a different syntax) are the [R notebooks written in RMarkdown](https://bookdown.org/yihui/rmarkdown/notebook.html) that are particularly prevalent for R-specific projects.

Reviewer: ```suggestion
Jupyter is one type of computational notebooks. Another type that shares the same philosophy and overall structure (but has a different syntax) are the [R notebooks written in RMarkdown](https://bookdown.org/yihui/rmarkdown/notebook.html).
```
